src/Functions/invPolyArcLength3.m

```diff
@@ -1,17 +1,27 @@
-%% invPolyArcLength3 - Numerically calculate the parameter (t) of parameterized [0,1] -> R^3 polynomial
+function t = invPolyArcLength3(polyCoeff, arcLength)
+% Numerically calculate the parameter (t) of parameterized [0,1] -> R^3 polynomial
 % *given* an arc length (measured from 0) i.e. find the corresponding t to an arc length
-% (e.g. given in [mm] from the origin) or in other words numerically invert polyArcLength3
+% (e.g. given in [mm] from the origin), or in other words numerically invert polyArcLength3
 %
-% Andreas Husch
-% Centre Hospitalier de Luxembourg, Dep. of Neurosurgery /
-% University of Luxembourg - Luxembourg Centre for Systems Biomedicine
-% 2016 - 2017
-% mail@andreashusch.de, husch.andreas@chl.lu
-function t = invPolyArcLength3(polyCoeff, arcLength)
-t = zeros(size(arcLength)); % default t=0 (i.e. in caes arcLength is 0)
+% USAGE:
+%
+%    t = invPolyArcLength3(polyCoeff, arcLength)
+%
+% INPUTS: 
+%    polyCoeff:     Coefficient matrix
+%    arcLength:     
```

Comment:
@adhusch Could you please provide a description of the inputs and outputs?

Reply:
arcLength: arc length in mm (as stated above)

Reply:
Oh right, thank you very much.